src/pages/ClerkScreen.tsx

```diff
@@ -23,80 +23,103 @@ import Screen from '../components/Screen'
 
 interface Props {
   appMode: AppMode
+  appPrecinctId: string
   ballotsPrintedCount: number
   election: OptionalElection
   isLiveMode: boolean
   fetchElection: VoidFunction
   isFetchingElection: boolean
-  setAppMode: InputEventFunction
+  setAppMode: (appModeName: AppModeNames) => void
+  setAppPrecinctId: (appPrecinctId: string) => void
   toggleLiveMode: VoidFunction
   unconfigure: VoidFunction
 }
 
 const ClerkScreen = ({
   appMode,
+  appPrecinctId,
   ballotsPrintedCount,
   election,
   isLiveMode,
   fetchElection,
   isFetchingElection,
   setAppMode,
+  setAppPrecinctId,
   toggleLiveMode,
   unconfigure,
 }: Props) => {
-  const [isModalOpen, setIsModalOpen] = useState(false)
-  const showModal = () => setIsModalOpen(true)
-  const hideModal = () => setIsModalOpen(false)
-  const handleToggleLiveMode = () => {
-    hideModal()
-    toggleLiveMode()
+  const changeAppMode: EventTargetFunction = event => {
+    const currentTarget = event.currentTarget as HTMLInputElement
+    const appModeName = currentTarget.dataset.appMode as AppModeNames
+    setAppMode(appModeName)
   }
-  const loadElection = () => {
-    fetchElection()
+
+  const changeAppPrecinctId: EventTargetFunction = event => {
+    const currentTarget = event.currentTarget as HTMLInputElement
+    const appPrecinctId = currentTarget.value
+    setAppPrecinctId(appPrecinctId)
   }
+
   const [isTestDeck, setIsTestDeck] = useState(false)
   const showTestDeck = () => setIsTestDeck(true)
   const hideTestDeck = () => setIsTestDeck(false)
   if (isTestDeck && election) {
     return (
       <TestBallotDeckScreen
         appName={appMode.name}
+        appPrecinctId={appPrecinctId}
         election={election}
         hideTestDeck={hideTestDeck}
         isLiveMode={false} // always false for Test Mode
       />
     )
   }
-  const isTestDecksAvailable = isLiveMode || (!isLiveMode && appMode.isVxPrint)
+  const isTestDecksAvailable = !isLiveMode && appMode.isVxPrint
```

Comment:
what about `VxMarkPrint`? Should test decks be available then too?

Reply:
ah, maybe that flag encompasses both?

Reply:
Yes, `VxMark+Print` includes `VxPrint`. Potentially confusing naming.